legal-api/src/legal_api/services/filings/validations/incorporation_application.py

```diff
@@ -69,3 +77,67 @@ def validate_offices(incorporation_json) -> Error:
         return msg
 
     return None
+
+
+def validate_roles(incorporation_json) -> Error:
+    """Validate the required completing party of the incorporation filing."""
+    parties_array = incorporation_json['filing']['incorporationApplication']['parties']
+    msg = []
+    completing_party_count = 0
+    incorporator_count = 0
+    director_count = 0
+
+    for item in parties_array:
+        role_array = item['roles']
+
+        if role_array.count('Completing Party') == 1:
```

Comment:
If the purpose is to check whether the role array has completing party it can be done like
if 'Completing Party' in role_array:

Reply:
oh nice, i like that much better! Will update for all roles 👍